src/pages/guidelines/pictograms/contribute.mdx

```diff
@@ -0,0 +1,94 @@
+---
+title: Pictograms
+description: Pictograms are visual symbols used to represent ideas, objects, or narratives at a glance. They work well in presentations and marketing communications.
+tabs: ['Library', 'Usage', 'Contribute']
+---
+
+<PageDescription>
+
+IBM’s pictograms are visual symbols used to represent ideas, objects, or
```

Comment:
```suggestion
Pictograms are visual symbols used to represent ideas, objects, or
```

Reply:
This content is copied from the IBM design language website so we'll need to make changes there.